### PR TITLE
003 header

### PR DESCRIPTION
**Pull request by FrancoisSalmon**

### CODE REVIEW

**Review by Gonaline — DISMISSED** [review]
ok

File: frontend/src/components/Archi/Header/style.jsx [hunk 1]
```diff
@@ -1,7 +1,48 @@
...
+    background-size: cover;
+    padding-top: 15vh;
+    width: 100vw;
+    height: 45vh;
+  }
+  .logoDesktop {
+    left: 0;
+    position: absolute;
+    position: fixed;
+    z-index: 1;
```
Comment (LuigiMenez) [inline frontend/src/components/Archi/Header/style.jsx:20]: Learn how to use a z-index

File: frontend/src/components/Archi/Header/style.jsx [hunk 2]
```diff
@@ -1,7 +1,48 @@
...
+    align-items: center;
+    height: 30vh;
+
+    .visible {
+      display: none;
+    }
+    .logoMobile {
+      display: block;
+      position: absolute;
+      z-index: 1;
```
Comment (LuigiMenez) [inline frontend/src/components/Archi/Header/style.jsx:38]: Learn how to use a z-index

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Learn how to use a z-index

**Review by Gonaline — DISMISSED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by Gonaline — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Pull request merged**